test-sbt/jvm/src/main/scala/zio/test/sbt/ZTestRunnerJVM.scala

```diff
@@ -84,8 +87,10 @@ final class ZTestRunnerJVM(val args: Array[String], val remoteArgs: Array[String
     defs: Array[TaskDef],
     console: zio.Console
   )(implicit trace: Trace): Array[ZTestTask[ExecutionEventSink]] = {
-    val testArgs        = TestArgs.parse(args)
-    val sharedSinkLayer = sinkLayer(console, ConsoleEventRenderer)
+    val testArgs = TestArgs.parse(args)
+
+    renderer = testArgs.testRenderer // Ensures summary is pretty in same style as rest of the test output
```

Comment:
@hmemcpy I think this obviates the need for `TestArgs.printSummary`, at least for some cases.